app/src/main/java/org/simple/clinic/teleconsultlog/success/TeleConsultSuccessEffect.kt

```diff
@@ -7,4 +7,6 @@ sealed class TeleConsultSuccessEffect {
   data class LoadPatientDetails(val patientUuid: UUID) : TeleConsultSuccessEffect()
 
   object GoToHomeScreen : TeleConsultSuccessEffect()
+
+  object GoToPrescriptionScreen : TeleConsultSuccessEffect()
```

Comment:
I am not entirely sure an `object` is the right kind to use here since the prescription screen will need to know which patient to load prescriptions for.